pkg/controllers/interruption/events/events.go

```diff
@@ -38,6 +38,15 @@ func SpotInterrupted(node *corev1.Node, nodeClaim *karpv1.NodeClaim) (evts []eve
 			DedupeValues:   []string{string(node.UID)},
 		})
 	}
+	for _, pod := range pods {
+		evts = append(evts, events.Event{
+			InvolvedObject: pod,
+			Type:           corev1.EventTypeWarning,
+			Reason:         "SpotInterrupted",
+			Message:        "Spot interruption warning was triggered",
```

Comment:
should this say something like "Pod will be evicted due to spot interruption warning" to be clearer ?